src/exe/Main.hs

```diff
@@ -125,7 +134,21 @@ optionDescriptors =
     (  "Optional. Path to output directory.\n"
     ++ "Prints to the console by default."
     )
+  , Option
+    ['f']
+    ["frontend"]
+    (ReqArg (\f opts -> opts { optFrontend = parseFrontend f }) "FRONTEND")
+    (  "Optional. Frontend to use for input parsing.\n"
+    ++ "Uses haskell-src-extensions by default"
+    )
   ]
+ where
+  -- | Parses a given string to one of the frontents. Returns @HSE@ by
+  --   default.
+  parseFrontend :: String -> Frontend
+  parseFrontend s = if map toLower s == "ghclib" || map toLower s == "ghc-lib"
```

Comment:
```suggestion
  parseFrontend s = if elem (map toLower s) ["ghclib", "ghc-lib", "ghc"]
```

Reply:
Actually, I think that we shouldn't allow alternative spellings for the front ends at all. They will just confuse users and make the help message, README and code more complicated than they need to be.

Reply:
I think it can be useful to allow alternative spellings. I don't think it is confusing for users because they can read in the README.md which spellings are accepted. I don't think writing the allowed spellings will make the README.md or help message more complicated. I also don't see why error messages would get more complicated.

Reply:
See https://github.com/FreeProving/haskell-src-transformations/pull/35#discussion_r456322799.